code/modules/mob/living/death.dm

```diff
@@ -35,6 +35,12 @@
 	flick(flick_name, animate)
 
 /mob/living/death(gibbed)
+	if(client && ismob(lastattacker)) //If they were murdered, increase the murderer's client kill count! We CoD now.
+		var/mob/M = lastattacker
+		if(M.client)
+			M.client.murders++ //Your LOVE increased.

```

Comment:
with the probably quite literally 1500 ways to kill people in SS13 there is no way we can track who killed who without manual log searching.